script.js

```diff
@@ -1,4 +1,146 @@
+  } else if (input == "stone" && computerChoice == "scissors") {
+    wins += 1;
+    myOutputValue = "You win!";
+  } else if (computerChoice == input) {
+    myOutputValue = "DRAW!!";
+  } else myOutputValue = "You lose...";
+
+  // Add emoji
+  if (computerChoice == "scissors") {
+    computerChoice = computerChoice + " ✂️";
+  } else if (computerChoice == "paper") {
+    computerChoice = computerChoice + " 🧻";
+  } else {
+    computerChoice = computerChoice + " 🪨";
+  }
+
+  if (input !== "scissors" && input !== "paper" && input !== "stone") {
+    return "Oops, please enter a valid choice (scissors / paper / stone)";
+  }
+  // add attempt count if valid input
+  attempts += 1;
+
+  if (gamemode == "normal") {
+    myOutputValue =
+      myOutputValue +
+      ` The computer used ${computerChoice}` +
+      `<br><br> You have played ${attempts} game(s) and won ${wins} time(s)` +
+      "<br><br> Enter (Scissors/Paper/Stone) to play another round (remember to change the gamemode if needed)!";
+    return myOutputValue;
+  } else {
+    myOutputValue =
+      myOutputValue +
+      ` The computer used ${input}` +
+      `<br><br> You have played ${attempts} game(s) and won ${wins} time(s)` +
+      "<br><br> Enter (Scissors/Paper/Stone) to play another round (remember to change the gamemode if needed)!";
+    return myOutputValue;
+  }
+};
+
+var koreanGame = function (input, computerChoice) {
+  if (input == "scissors" && computerChoice == "paper") {
+    wins += 1;
+    myOutputValue = "You win!";
+  } else if (input == "paper" && computerChoice == "stone") {
+    wins += 1;
+    myOutputValue = "You win!";
+  } else if (input == "stone" && computerChoice == "scissors") {
+    wins += 1;
+    myOutputValue = "You win!";
+  } else if (computerChoice == input) {
+    myOutputValue = "DRAW!!";
+  } else myOutputValue = "You lose...";
+
+  console.log(input);
+  console.log(computerChoice);
+  console.log(myOutputValue);
+  console.log(currentWinner);
+  if (myOutputValue == "You win!") {
+    currentWinner = "you";
+    return (
+      "MUK-JJI-PPA!! <br><br>" +
+      `You used ${input} and computer used ${computerChoice}` +
+      "<br><br> Quickly enter another choice of either scissors, paper or stone! <br><br>" +
+      `Current Winner: ${currentWinner}`
+    );
+  }
+  if (myOutputValue == "You lose...") {
+    currentWinner = "computer";
+    return (
+      "MUK-JJI-PPA!!<br><br>" +
+      `You used ${input} and computer used ${computerChoice}` +
+      "<br><br> Quickly enter another choice of either scissors, paper or stone! <br><br>" +
+      `Current Winner: ${currentWinner}`
+    );
+  } else {
+    if (currentWinner == "") {
+      return `Draw! You used ${input} and computer used ${computerChoice}`;
+    } else {
+      var koreanMessage =
+        "We have a winner! <br><br>" +
+        `You used ${input} and the computer used ${computerChoice}, so the ultimate winner is ${currentWinner}!!!`;
+      currentWinner = "";
```

Comment:
Were you trying to restart the game here, by setting winner name back to an empty string? 
A suggestion would be to encapsulate all the things that need to be reset in a function, so it'll be clearer what you are trying to say through your code